lib/inventoryware/utils.rb

```diff
@@ -75,12 +76,16 @@ def self.find_file(search_val, dir)
       return results
     end
 
+    def self.prompt
+      @prompt ||= TTY::Prompt.new
+    end
+
     def self.get_new_asset_type
-      type = ''
-      while type.empty?
-        type = TTY::Prompt.new.ask('Enter the type of the new assets being created')
+      prompt.select('Select an asset type?') do |menu|
```

Comment:
This isn't a question.  Can we replace the `?` with a `.`.

Reply:
https://github.com/openflighthpc/flight-inventory/pull/156/commits/20eb479e9639d250c9433f59d0d843d3f8c03c20